src/main/java/com/uber/profiling/profilers/CpuAndMemoryProfiler.java

```diff
@@ -198,9 +207,13 @@ public synchronized void profile() {
 
         map.put("heapMemoryTotalUsed", heapMemoryTotalUsed);
         map.put("heapMemoryCommitted", heapMemoryCommitted);
+        map.put("heapMemoryMax", heapMemoryMax);
+        map.put("heapMemoryInit", heapMemoryInit);
```

Comment:
heapMemoryInit is a fixed value, and does not change during jvm process running, would suggest remove it from the metrics